src/pages/BookPage/index.js

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { selectBookDetails } from "../../store/bookPage/selectors";
+import {
+  fetchBookDetails,
+  setBookDetails,
+  addNewReview,
+  updateAmountOfLikes,
+} from "../../store/bookPage/actions";
+import { selectToken, selectUser } from "../../store/user/selectors";
+import Form from "react-bootstrap/Form";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+import { Col } from "react-bootstrap";
+import "./index.css";
+
+export default function BookPage() {
+  const dispatch = useDispatch();
+  let { collectionId, bookId } = useParams();
+  const bookDetails = useSelector(selectBookDetails);
+  const user = useSelector(selectUser);
+  const token = useSelector(selectToken);
+  const [reviewTitle, setReviewTitle] = useState("");
+  const [reviewContent, setReviewContent] = useState("");
+
+  useEffect(() => {
+    dispatch(fetchBookDetails(collectionId, bookId));
+  }, [dispatch, setBookDetails]);
+
+  function submitForm(event) {
+    event.preventDefault();
+
+    const newReviewData = {
+      reviewTitle: reviewTitle,
+      reviewContent: reviewContent,
+      collectionId: collectionId,
+      bookId: bookId,
+    };
```

Comment:
no need to repeat the key/value in an object if you're going to call the keys the same as the constants are called. this works:
```
const newReviewdata = { reviewTitle, reviewContent, collectionId, bookId }
```

Reply:
oh, yes, the same in backend, thanks